vnext/package.json

```diff
@@ -92,7 +92,8 @@
       "major",
       "minor",
       "patch"
-    ]
+    ],
+    "gitTags": true
```

Comment:
This was unintentionally removed with https://github.com/microsoft/react-native-windows/commit/edf2bf35d8cba11571bb5938fc0f37044e5606d5#r78262809 at a point after the 0.69 cut. As-is it would cause 0.70 not to correctly create tags or release notes.